docs/content/mobile/android/setup.md

```diff
@@ -7,17 +7,17 @@
        * The Android SDK
        * The Android Studio emulator
     * [Android NDK r13b](https://developer.android.com/ndk/downloads/older_releases) - Extract it to a directory of your choice. Note down this directory path as you will be needing it in the following steps.
-    * [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+    * [Java Platform, Standard Edition Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) - download and install the latest version.
   * (Optional) [Unity Remote](https://play.google.com/store/apps/details?id=com.unity3d.genericremote) - this is Unity’s solution for faster development iteration times.
 
 ## Set up your Unity Editor
 After installing these dependencies, open the Unity project in `<path-to-your-project>/workers/unity`.
 
   1. In the Unity Editor, go to **File** > **Build Settings**. Select **Android** and then click on **Switch Platform**.
-  1. In the Unity Editor, go to **Edit** > **Preferences**. In the **External Tools** window, add the file paths to the tools in the **Android** section:
+  1. In the Unity Editor, go to **Edit** > **Preferences**. In the **External Tools** window, in the Android section, input the paths to the SDK, JDK and NDK. The easiest way to ensure that Unity reads the filepath correctly is to use the browse option:
```

Comment:
can we add a note that they only need to download the NDK and add the path, if they want to build Android using IL2CPP?

Reply:
![capture](https://user-images.githubusercontent.com/19690292/52214807-754ba380-288a-11e9-8904-7e822241c034.PNG)

Fixed.

Reply:
If "Android" is on the UI, it's bolded.